Tests/LibWeb/Text/input/navigation/ismap-img-sends-query-string-from-link.html

```diff
@@ -0,0 +1,22 @@
+<script src="../include.js"></script>
+<script>
+    asyncTest(async done => {
+        const iframe = document.getElementById("testIframe");
+
+        let count = 0;
+        function handleMessage(event) {
+            ++count;
+            println(`Search query from iframe: #${count} = '${new URL(event.data).search}'`);
+            if (count === 1) {
+                internals.click(50, 50);
+            } else if (count === 2) {
+                done();
+            }
+        }
+
+        window.addEventListener("message", handleMessage);
+
```

Comment:
if you run `prettier` on the .html files, it'll clean up extra whitespace like this